Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -39,9 +39,11 @@
    "source": [
     "# Uncomment & specify proxy value if applicable\n",
     "# %env https_proxy=http://<proxy_host>:<port>\n",
+    "# After default.svc append comma separated list of IP address, hostname, CIDR block, service name, etc.\n",
+    "# %env no_proxy=\"localhost,.cluster.local,.svc,.default.svc, <host>:<port>, x.x.x.x/y,...\"\n",
     "# Install required packages (Kubeflow Pipelines and Katib SDK).\n",
     "import sys\n",
-    "!{sys.executable} -m pip install \"pyarrow>7.0.0\" \"kfp>=1.8.0,<=1.8.22\" kubeflow-katib==0.16.0"
+    "!{sys.executable} -m pip install kfp[kubernetes]"
```

Comment:
Do we need to install it actually? If we do not install `kfp[kubernetes]`, then example will fail, right? Looks strange, I've thought that we install the whole kfp package. If example is not working without it, I'm ok to leave it here. But in future in notebooks we can install this in all notebook images I think

Reply:
We can add it in the notebooks in future. Have tested it without installing kfp[kubernetes] it was failing as we are calling it for creating the PVC.

Reply:
@roopak-rastogi thank you, got it, okay. It is fine

Reply:
Now, due to issue with `storageClassName` we are creating pvc with `kubectl`, so `kfp.kubernetes` module is not needed any more.